api/resolvers/index.js

```diff
@@ -393,6 +393,19 @@ const resolvers = {
         isApproved: true,
       }).save();
 
+      customField = {
```

Comment:
This is an undefined variable, should have `const` in front of it

Reply:
You could add this object in the first `new Event` call on line 379, like this:

```javascript
  const event = await new Event({
        slug,
        title,
        description,
        summary,
        currency,
        registrationPolicy,
        organizationId: currentOrg.id,
        customFields: [
          {
            name: "Description",
            description: "Describe your Dream",
            type: "MULTILINE_TEXT",
            isRequired: false,
            isShownOnFrontPage: false,
            position: 1001,
          },
        ],
      }).save();
```

and avoid having to save the event twice